src/editors/containers/TextEditor/components/ImageSettingsModal/hooks.js

```diff
@@ -157,8 +158,20 @@ export const dimensionHooks = () => {
     lock,
     unlock,
     value: local,
-    setHeight: (height) => setLocal({ ...local, height: parseInt(height, 10) }),
-    setWidth: (width) => setLocal({ ...local, width: parseInt(width, 10) }),
+    setHeight: (height) => {
```

Comment:
I think maybe we should set some kind of[ feedback](https://paragon-openedx.netlify.app/components/form/form-control-feedback/) or error if the user puts in things other that numbers and the percent sign in because right now an acceptable input is "connor%"

Reply:
Although I'm not super keen on not providing any feedback when the user enters in Connor%, we were told to not make any UI changes, so I guess we will live with that.